IntegrationTests/common/script/CompareMemPrintsFW.py

```diff
@@ -182,31 +221,105 @@ def compare(comparison_filename="", fail_on_error=False, file_location='./', pre
-                               reference_filenames=[("../../../emData/MemPrints/Matches/CandidateMatches_CM_L3PHIC%i_04.dat" % (i))], save=args.save, verbose=args.verbose)
+    if (args.predefined == "PRMEMC"):
+        if os.path.exists('./dataOut/AP_L3PHIC.txt'):
+            ret_sum += compare(comparison_filename="./dataOut/AP_L3PHIC.txt", fail_on_error=False, file_location=args.file_location, predefined=args.predefined,
+                               reference_filenames=["../../../emData/MemPrints/TrackletProjections/AllProj_AP_L3PHIC_04.dat"], save=args.save, verbose=args.verbose)
+        
+        for i in range(17,25):
+            if os.path.exists(('./dataOut/VMPROJ_L3PHIC%i.txt' % (i))):
+                ret_sum += compare(comparison_filename=("./dataOut/VMPROJ_L3PHIC%i.txt" % (i)), fail_on_error=False, file_location=args.file_location, predefined=args.predefined,
+                                   reference_filenames=[("../../../emData/MemPrints/VMProjections/VMProjections_VMPROJ_L3PHIC%i_04.dat" % (i))], save=args.save, verbose=args.verbose)
+        
+        for i in range(17,25):
+            if os.path.exists(('./dataOut/CM_L3PHIC%i.txt' % (i))):
+                ret_sum += compare(comparison_filename=("./dataOut/CM_L3PHIC%i.txt" % (i)), fail_on_error=False, file_location=args.file_location, predefined=args.predefined,
+                                   reference_filenames=[("../../../emData/MemPrints/Matches/CandidateMatches_CM_L3PHIC%i_04.dat" % (i))], save=args.save, verbose=args.verbose)
+        
+        ret_sum += compare(comparison_filename="./dataOut/FM_L1L2_L3PHIC.txt", fail_on_error=False, file_location=args.file_location, predefined=args.predefined,
+                           reference_filenames=["../../../emData/MemPrints/Matches/FullMatches_FM_L1L2_L3PHIC_04.dat"], save=args.save, verbose=args.verbose)
+        ret_sum += compare(comparison_filename="./dataOut/FM_L5L6_L3PHIC.txt", fail_on_error=False, file_location=args.file_location, predefined=args.predefined,
+                           reference_filenames=["../../../emData/MemPrints/Matches/FullMatches_FM_L5L6_L3PHIC_04.dat"], save=args.save, verbose=args.verbose)
+
+    elif (args.predefined == "IRVMR"):
+        # IL
+        for i in ['L', 'D']:
+            for j in ['A', 'B', 'C', 'D']:
+                if os.path.exists(('./dataOut/IL_%c2PHI%c_PS10G_3_A.txt' % (i,j))):
+                    ret_sum += compare(comparison_filename=("./dataOut/IL_%c2PHI%c_PS10G_3_A.txt" % (i,j)), fail_on_error=False, file_location=args.file_location, predefined=args.predefined,
+                                       reference_filenames=[("../../../emData/MemPrints/InputStubs/InputStubs_IL_%c2PHI%c_PS10G_3_A_04.dat" % (i,j))], save=args.save, verbose=args.verbose)
+        # AllStubs - temporarily removed due to issues with not having nentries port
```

Comment:
We need a way to recover the comparison for the AllStub memory.

Reply:
The allstub comparison is back now, although this increased the number of errors from 74 to 614.